Sources/OpenCombine/Published.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Published.swift
+//  OpenCombine
+//
+//  Created by Евгений Богомолов on 01/09/2019.
+//
+
+#if swift(>=5.1)
+@propertyWrapper public struct Published<Value> {
+
+    /// Initialize the storage of the Published
+    /// property as well as the corresponding `Publisher`.
+    public init(initialValue: Value) {
+        _value = initialValue
+    }
+
+    public struct Publisher: OpenCombine.Publisher {
+
+        /// The kind of values published by this publisher.
+        public typealias Output = Value
+
+        /// The kind of errors this publisher might publish.
+        ///
+        /// Use `Never` if this `Publisher` does not publish errors.
+        public typealias Failure = Never
+
+        /// This function is called to attach the specified
+        /// `Subscriber` to this `Publisher` by `subscribe(_:)`
+        ///
+        /// - SeeAlso: `subscribe(_:)`
+        /// - Parameters:
+        ///     - subscriber: The subscriber to attach to this `Publisher`.
+        ///                   once attached it can begin to receive values.
+        public func receive<SubscriberType>(subscriber: SubscriberType)
+            where Value == SubscriberType.Input,
+            SubscriberType: Subscriber,
+            SubscriberType.Failure == Published<Value>.Publisher.Failure
+        {
+            _passthrowObject.subscribe(subscriber)
+        }
+
+        private let _passthrowObject = OpenCombine.PassthroughSubject<Value, Never>()
+
+        internal func send(_ input: Output) {
```

Comment:
Instead of adding this new method you can just mark the subject `fileprivate` and use it directly from `Published`